packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -59,7 +60,32 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.twapLength,
       config.lookback,
       getTime,
-      config.invertPrice // Not checked in config because this parameter just defaults to false.
+      config.invertPrice, // Not checked in config because this parameter just defaults to false.
+      config.poolDecimals,
+      config.priceFeedDecimals // This defaults to 18 unless supplied by user
```

Comment:
why is this showing up as a diff? I'm pretty sure this is what the actual `UniswapPriceFeed` looks like.

Reply:
My guess is that there are some merge issues, where these changes accidentally got removed in the merge process, but then were re-added in this branch to get the tests to pass.